include/deal.II/grid/cell_id.h

```diff
@@ -317,6 +323,24 @@ CellId::operator<(const CellId &other) const
 }
 
 
+
+inline bool
+CellId::is_parent_of(const CellId &other) const
+{
+  if (this->coarse_cell_id != other.coarse_cell_id)
+    return false;
+
+  if (n_child_indices + 1 != other.n_child_indices)
+    return false;
+
+  for (auto idx = 0u; idx < n_child_indices; ++idx)
```

Comment:
Let's be explicit with the type here:
```suggestion
  for (unsigned int idx = 0; idx < n_child_indices; ++idx)
```